bin/metrics_exporter.pl

```diff
@@ -0,0 +1,30 @@
+use HTFeed::JobMetrics;
```

Comment:
I would expect to see `use strict;` and `use warnings;` at the top of every perl script, OR a comment with a compelling argument why not.